addons/mail/static/src/messaging/entity/messaging/messaging.js

```diff
@@ -186,6 +185,7 @@ function MessagingFactory({ Entity }) {
             default: 0,
         }),
         partnerRoot: one2one('Partner'),
+        publicPartner: one2one('Partner'),
```

Comment:
No reason to enforce those to be one2one, many2one should work fine and would be a preferable default